launchable/commands/record/tests.py

```diff
@@ -604,26 +604,32 @@ def get_record_start_at(session: Optional[str], client: LaunchableClient):
 
     sub_path = "builds/{}".format(build_name)
 
-    res = client.request("get", sub_path)
-    if res.status_code != 200:
-        if res.status_code == 404:
-            msg = "Build {} was not found. " \
-                  "Make sure to run `launchable record build --name {}` before `launchable record tests`".format(
-                      build_name, build_name)
-        else:
-            msg = "Unable to determine the timestamp of the build {}. HTTP response code was {}".format(
-                build_name,
-                res.status_code)
-        click.echo(click.style(msg, 'yellow'), err=True)
-
-        # to avoid stop report command
+    try:
+        res = client.request("get", sub_path)
+        if res.status_code != 200:
+            if res.status_code == 404:
+                msg = "Build {} was not found. " \
+                    "Make sure to run `launchable record build --name {}` before `launchable record tests`".format(
+                        build_name, build_name)
+            else:
+                msg = "Unable to determine the timestamp of the build {}. HTTP response code was {}".format(
+                    build_name,
+                    res.status_code)
+            click.echo(click.style(msg, 'yellow'), err=True)
+
+            # to avoid stop report command
+            return INVALID_TIMESTAMP
+        created_at = res.json()["createdAt"]
+        Logger().debug("Build {} timestamp = {}".format(build_name, created_at))
+        t = parse_launchable_timeformat(created_at)
+        return t
+    except Exception as e:
+        tracking_client.send_error_event(
+            event_name=Tracking.ErrorEvent.INTERNAL_CLI_ERROR,
+            stack_trace=str(e),
+        )
         return INVALID_TIMESTAMP
```

Comment:
What I worry is that, if we introduce any mistakes within the try block, say a simple coding mistake, we'd never notice because this silently recovers with `INVALID_TIMESTAMP`.

I get the value that, during the production customer use, the up to date check is not valuable enough to stop & hold the delivery pipeline, so we should recover from that error.

Can we meet both of those goals instead of sacrificing one over the other?